Formula/h/helidon.rb

```diff
@@ -0,0 +1,23 @@
+class Helidon < Formula
+  desc "Command-line tool for Helidon application development"
+  homepage "https://helidon.io/"
+  url "https://github.com/helidon-io/helidon-build-tools/archive/refs/tags/3.0.6.tar.gz"
+  sha256 "749cf3fd162bb9449ab57584c0bdf8874114d678499071ea522c047637de0f90"
+  license "Apache-2.0"
+
+  depends_on "maven"
+  depends_on "openjdk"
+
+  def install
+    system "mvn", "package", "-f", "cli/impl/pom.xml", "-DskipTests"
+    system "unzip", "cli/impl/target/helidon-cli"
+    libexec.install "helidon-3.0.6/bin", "helidon-3.0.6/helidon-cli.jar", "helidon-3.0.6/libs"
+    (bin/"helidon").write_env_script libexec/"bin/helidon", Language::Java.overridable_java_home_env
+  end
+
+  test do
+    ENV["JAVA_HOME"] = Formula["openjdk"].opt_prefix
```

Comment:
Given the install block, this should not be needed

Reply:
Then you need to check what bin/helidon has configured after the install because that should not happen

Reply:
Fixed it. Seems like a a job is hanging, are you able to re trigger the job or should I squash the commit and force push to re run all the jobs ?